src/serialization/string_view.hpp

```diff
@@ -28,6 +28,7 @@ that class. */
 
 namespace utils {
 using string_view = boost::string_view;
+typedef boost::basic_string_view<uint8_t, std::char_traits<uint8_t>> u8string_view;
```

Comment:
This type name is confusing. It suggests "UTF-8 string" instead of "uint8_t string".

I suggest renaming it to `byte_string_view`.

Reply:
I agree, my first thought on seeing that was indeed UTF-8. In fact I'm pretty sure there's an actual `u8string` which was intended for UTF-8 strings? Or at least something very similar (`u16string`?).